code/game/objects/items/stacks/sheets/organic/cloths_recipes.dm

```diff
@@ -2,6 +2,7 @@
 
 GLOBAL_LIST_INIT(cloth_recipes, list ( \
 	new/datum/stack_recipe("white jumpsuit",						/obj/item/clothing/under/color/white, 3, time = 4 SECONDS), \
+	new/datum/stack_recipe("plush fabric",						/obj/item/toy/empty_plush, 5, time = 4 SECONDS), \
```

Comment:
It's not with "clothes" it's with "cloth", as in the recipes accessible from using that stack in hand? You'll notice it includes bags and canvas too for that reason. I don't understand the issue here?

Reply:
It's not the fact that its in the cloth recipes, it's the positioning in the cloth recipe list. It is placed with the cloth recipe for clothes rather than for general cloth based materials.